public/app/core/components/DynamicImports/SafeDynamicImport.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Loadable from 'react-loadable';
+import { css } from 'emotion';
+import { LoadingChunkPlaceHolder } from './LoadingChunkPlaceHolder';
+import { ErrorLoadingChunk } from './ErrorLoadingChunk';
+
+function getAlertPageStyle(): string {
+  return css`
+    width: 508px;
+    margin: 128px auto;
+  `;
+}
+
+export const Loading = (props: { error: Error; pastDelay: boolean }) => {
+  const { error, pastDelay } = props;
+
+  if (error) {
```

Comment:
So, now this will catch only the chunk loading errors,while error boundary will catch all the other ones, right?

Reply:
Yes, this will catch any errors during loading of component.